### DISCUSSION TITLE
Could I upload image to invoke an image recognize application?

### DISCUSSION THREAD

**Discussion by archichen** [Hydrospheredata/hydro-serving#333] [Q&A]

**Comment by Valenzione** [comment]
Hi, since it's not an issue I'll convert this to a new discussion 😉

**Reply by archichen** [reply]
wow, cool, this may new way to discuss on github

**Comment by tidylobster** [comment]
HI @archichen. You can submit images as tensors to your models. The conversion to tensor is typically done on the client's side and the resulting tensor is sent for prediction.

**Reply by archichen** [reply]
Thank you for reply! But I'm not an AI engineer and have not seen related content on the Hydrosphere website... So that I do not understand that how to convert image to tensor and send to model. Could you provide document or example? Such as how to package images and how to invoke.

**Reply by tidylobster** [reply]
At the moment there aren't' any tutorials on image processing in the documentation. But we have a few [examples](https://github.com/Hydrospheredata/hydro-serving-example/tree/master/examples) of processing MNIST images, although they are outdated. I'll make a quick draft with the updated examples and share it here.

**Reply by tidylobster** [reply]
Here, you can check this draft for the latest working image example: https://github.com/Hydrospheredata/hydro-serving-example/pull/51. If you have any questions, I'm happy to help.
